packages/next/src/routes/rest/buildFormState.ts

```diff
@@ -24,7 +24,33 @@ export const getFieldSchemaMap = (config: SanitizedConfig): FieldSchemaMap => {
 export const buildFormState = async ({ req }: { req: PayloadRequest }) => {
   const reqData: BuildFormStateArgs = req.data as BuildFormStateArgs
 
-  // TODO: run ADMIN access control for user
+  const incomingUserSlug = req.user?.collection
+  const adminUserSlug = req.payload.config.admin.user
+
+  // If we have a user slug, test it against the functions
+  if (incomingUserSlug) {
+    const adminAccessFunction = req.payload.collections[incomingUserSlug].config.access?.admin
```

Comment:
Have you made sure that `canAccessAdmin` is not already on the `req` somewhere?

Reply:
@jacobsfletch that property is not on the request